decisions/20240313-try-w3up.md

```diff
@@ -0,0 +1,145 @@
+# nft.storage should try w3up for web3 storage of new uploads
+
+Authors
+- bengo@protocol.ai
+
+## Problem Statement
+
+nft.storage needs to offer web3 storage and currently does so using a homegrown solution that may be expensive to operate, maintain, and scale.
```

Comment:
It seems a little odd to not mention anywhere in this doc the reality that nft.storage's custom Filecoin pipeline (DAGCargo) is about to break for taking new data - so the experience explicitly will stop working. 

It feels like someone revisiting this ADR in a year might want to know this?

Reply:
> not mention anywhere in this doc the reality that nft.storage's custom Filecoin pipeline (DAGCargo) is about to break for taking new data

Do you have any ideas for what I could link to that publicly explains that reality?

Reply:
@hannahhoward I guess what I'm feeling like is missing is: Where is the ADR for the root cause of why dagcargo is about to break? For me that isn't a decision I'm making, it's a research task to go find out the root cause. I could do that instead of implementing this plan, but I don't think we'll regret doing this plan as described this sprint no matter what.

Reply:
I mentioned dagcargo briefly. Thanks for the suggestion. https://github.com/nftstorage/nft.storage/pull/2518/commits/dc2bf27f8599038d886ad1d43a4795a6f4102112